src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile.cs

```diff
@@ -323,6 +323,22 @@ public async Task When_CreateFile_GenerateUnique_Exists_Folder()
 			}
 		}
 
+		[TestMethod]
+		public async Task When_Open_By_Encoded_URI_With_Space()
+		{ 
+			var uri = new Uri($"ms-appx:///Assets/Arrow Back.svg"); 
```

Comment:
Let's use an explicit name for the asset, so it does not get conflated with something else:

```suggestion
			var uri = new Uri($"ms-appx:///Assets/Asset With Spaces.svg"); 
```

Reply:
Ok!